### PR TITLE
LPS-144182

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-144182

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js [hunk 1]
```diff
@@ -37,16 +37,14 @@ describe('DefaultPage', () => {
 		expect(getByText('Page title')).toBeInTheDocument();
 	});
 
-	it('shows submit again button when Limit to one submission per user is disabled', () => {
-		const {getByText} = render(<DefaultPage />);
+	it('shows submit again button when the show submit again flag is disabled', () => {
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js:40]: @rodrigopaulino shouldn’t be '`shows submit again button when the show submit again flag is enabled`'?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js [hunk 2]
```diff
@@ -37,16 +37,14 @@ describe('DefaultPage', () => {
...
+		const {getByText} = render(<DefaultPage showSubmitAgainButton />);
 
 		expect(getByText('submit-again')).toBeInTheDocument();
 	});
 
-	it('hides submit again button when Limit to one submission per user is enabled', () => {
-		const {queryByText} = render(
-			<DefaultPage limitToOneSubmissionPerUser />
-		);
+	it('hides submit again button when the show submit again flag is enabled', () => {
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js:46]: @rodrigopaulino shouldn’t be '`hides submit again button when the show submit again flag is disabled`'?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js:40]: Oh boy. 🤭 It should. Thanks again, @carolmariaabb

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
Changes made, @carolmariaabb

**Review by carolmariaabb — APPROVED** [review]

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**